sale_triple_discount/models/sale_order_line.py

```diff
@@ -136,6 +135,6 @@ def triple_discount_postprocess(self, prev_values):
         Updating the cache provides consistency through recomputations."""
         self.invalidate_cache(
             fnames=['discount', 'discount2', 'discount3'],
-            ids=[l.id for l in prev_values.keys()])
-        for line, prev_vals_dict in prev_values.items():
+            ids=[l.id for l in list(prev_values.keys())])
+        for line, prev_vals_dict in list(prev_values.items()):
```

Comment:
this call to `list` is not needed

Reply:
ok @simahawk, `2to3` is the responsible of this change :wink: